Responsive design

VelzckC0D3 opened the pull request:
# Responsive Design

## _What I have submitted  ⬆️_
✔️ _Used a 768px media query in order to make the sections responsive._  `(Requested)`
✔️ _Succesfully implemented responsive design for "Works" & "About me" sections._ `(Requested)`
✔️ _Succesfully implemented a responsive design for "Skills" `(Requested)`_
✔️ _Succesfully implemented a responsive design for "Form" sections (with the help of my coding partner @zam-cham)_ `(Requested)`
✔️ _Checked linters_ `(Requested)`
✔️ _Flexbox, animations, style and it replicates the Microverse Template 2._ `(Requested)`

## ❗♻️ _Review Pull Request Changelog_  ♻️❗
✅ _As my first reviewers noticed, there was an inconvenience with some IMG because I added it like: `/style/...` but that could cause errors, everyone was changed to `style/...`_
✅ _Added the optional feature since I noticed that it was actually a good idea for the UI/UX of the website_
✅ _Refreshed the cache on the GIbash (untracked and tracked the files), in order to avoid any inconvenience with the files of the website

##  🚀✨ _Extra features!_ ✨🚀
✔️ _All the grids are animated with a carefully made animation!_ 
✔️ _Responsive and adaptative at the same time!_  
✔️ _Personalized hover animations for all buttons_  
✔️ _Personalized style without leaving the original template_ 

![ezgif com-crop](https://user-images.githubusercontent.com/92229666/220820100-329ab99d-92bb-4b4e-af6a-74ee964e8e2f.gif)
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/92229666/220820470-f2252955-c8e2-4c0a-8b6b-8c25676e551b.gif)


### **_I want to say thanks to my coding parter @zam-cham who helped me during this process_ 🫡**

# 🎈 _Thank you for your review!_ 🎈
![ThumbsupBatmanGIF](https://user-images.githubusercontent.com/92229666/220835080-ceafd08b-7c24-4d41-b2c2-a30c25ef7c14.gif)

VelzckC0D3 committed:
Reponsive header & main

Added responsive design for the following elements:
- Header: re organized and adapted according to the desktop device example
- Main section (title): re organized and adapted according to the desktop device example

VelzckC0D3 committed:
Responsive web design

- Implemented responsive design for "Works" section with carefully made CSS animations and styling
- Implemented  responsive  design for "About me" section
- Implemented responsive design for "Skills" section
- Implemented responsive design for the Form and the "Footer" of the website
- Double checked all the linters

VelzckC0D3 committed:
Readme Update

- Added the collaborator (my coding partner)

VelzckC0D3 committed:
Readme

Deleted the internal note about MIT License.

VelzckC0D3 committed:
Linters

- Double checked the linters: corrected

AbayomiOlaoye commented on index.html:
```diff
@@ -25,6 +25,17 @@
+                            <li class="liMenu"> <a class="liMenuLink" href="#aboutMe">About</a></li>
+                            <div id="menuContact">
+                                <li class="liMenu1"><a href="#form">Contact</a></li>
+                                <li id="liMenuIcon"><a href="#form"> <img src="/style/img/contact.svg" alt=""></a>
+                                </li>
```
- [x] Brilliant submission! 💪 Great Design 🥇 But it seems this icon is not propping up on my device and that might be due to the slight omission on `.` just before `/style.......`. Kindly reference the path well so that users won't miss out on the opportunity to see all what aesthetic designs you have for them to experience.

![image](https://user-images.githubusercontent.com/99715640/220823138-dded65d9-0f5a-4f0c-b48e-8a7ddc6f9177.png)

- [x] [OPTIONAL]_Here is a plus for this icon: It could be used with a `a` tag where it's `href` attribute is set to `mailito:example@check.ext`. Once user clicks on it, it pops up there email configs to send a mail to you. I think it broadens accessibility and smoothens user experience._

VelzckC0D3 replied:
Hi, @AbayomiOlaoye ,I didn't knew that if you put `/style....` you should add another folder "." to it, I deleted this error for each line that it was present (that were actually the ones that you were not able to see..) !! thank you for that lesson! 🤓

Also, I implemented the `mailto` to the message Icon, thanks ! I appreciate your code review 😁 and I did think as well that that's a good implementation for the UI/UX !!

AbayomiOlaoye replied:
@VelzckC0D3 That's good to hear 😃, and thanks for the kind words 😊.

AbayomiOlaoye commented on index.html:
```diff
@@ -262,59 +290,72 @@ <h2>About me</h2>
-                        <li class="skillsLi inter">CSS</li>
-                    </ul>
-                </div>
-                <div class="skillsCont flexColumn">
-                    <div class="sTitle inter">
-                        <h2><img src="style/icon/square.svg" alt="square icon"> Frameworks</h2>
+                    <div class="skillsCont flexColumn">
+                        <div class="sTitle inter">
+                            <h2><img src="style/icon/square.svg" alt="square icon"> Frameworks</h2>
+                        </div>
+                        <ul class="skillsUl">
+                            <li class="skillsLi inter">Bootstrap</li>
+                            <li class="skillsLi inter">Ruby on Rails</li>
+                            <li class="skillsLi inter">RSPec</li>
+                            <li class="skillsLi inter">CapyBara</li>
+                            <li class="skillsLi inter">Selenium</li>
+                        </ul>
                     </div>
-                    <ul class="skillsUl">
-                        <li class="skillsLi inter">Bootstrap</li>
-                        <li class="skillsLi inter">Ruby on Rails</li>
-                        <li class="skillsLi inter">RSPec</li>
-                        <li class="skillsLi inter">CapyBara</li>
-                        <li class="skillsLi inter">Selenium</li>
-                    </ul>
-                </div>
-                <div class="skillsCont flexColumn">
-                    <div class="sTitle inter">
-                        <h2><img src="/style/icon/circle.svg" alt="circle"> Skills</h2>
+                    <div class="skillsCont flexColumn">
+                        <div class="sTitle inter">
+                            <h2><img src="/style/icon/circle.svg" alt="circle"> Skills</h2>
+                        </div>
+                        <ul class="skillsUl">
+                            <li class="skillsLi inter">Codekit</li>
+                            <li class="skillsLi inter">GitHub</li>
+                            <li class="skillsLi inter">CodePen</li>
+                            <li class="skillsLi inter">GitLab</li>
+                            <li class="skillsLi inter">Terminal</li>
+                        </ul>
```
- [ ] You have a good eye for design 🤗 but I think you need to check why the background image for this section and the prepended icon to the left of `Skills` are not showing as your displayed in your descriptive pull request message.

![image](https://user-images.githubusercontent.com/99715640/220823865-a9975f86-d7df-4067-8889-ad475d40a557.png)

VelzckC0D3 replied:
Coorected!

AbayomiOlaoye reviewed (changes requested):
## STATUS: CHANGE REQUIRED ♻️ 

Hi @VelzckC0D3 👏 👋 , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

![so_close](https://user-images.githubusercontent.com/99715640/211950260-0e3881c0-9d4f-4ea2-a670-9ef45a23a0f5.gif)

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Nice UI implementation ✔️ 
- Awesome additional features ✔️ 

## Required Changes ♻️
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

VelzckC0D3 committed:
Request Errors Fix

- As the reviewer told me, there was an error on several lines, you can't reference the path starting with "/style" so I corrected it on each line were it was present
- Also, implemented the optional feature in order to offer a better UI/UX    (added "mailto" action to the card icon on the nav bar)
- Refreshed all the files cache on the console to avoid any missed information error

VelzckC0D3 committed:
Review Correction

Double checked the errors and find 2 more, corrected

GedeonTS reviewed (approved):
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @VelzckC0D3 ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"> 

## Highlights :clap: :tophat: 
- [x] No linter errors :+1: 
- [x] Good use of GitHub :+1: 
-  [x] The background images are displayed well :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
